plutus-streaming/src/Plutus/Streaming/ChainIndex.hs

```diff
@@ -0,0 +1,52 @@
+module Plutus.Streaming.ChainIndex (
```

Comment:
This module and the following (Plutus.Streaming.LedgerState) where basically proof-of-concept indexers. I don't think they belong to the streaming package.